gui_qt/litellm_worker.py

```
@@ -0,0 +1,111 @@
+
+
+class LiteLLMVersionWorker(QThread):
+    completed = Signal(str, str, object)
+
+    def run(self) -> None:
+        installed = installed_litellm_version()
+        try:
+            request = Request(
+                LITELLM_PYPI_URL,
+                headers={"User-Agent": "renpy-translation-lab"},
+            )
```

Ruff (0.15.20)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)